docs/quick-start.md

```diff
@@ -3,7 +3,7 @@ hide:
   - navigation
 ---
 
-Try running a Tart VM on your Apple Silicon device running macOS 12.0 (Monterey) or later (will download a 25 GB image):
+Try running a Tart VM on your Apple Silicon device running macOS 12.0 (Ventura) or later (will download a 25 GB image):
```

Comment:
Perhaps this should be left as is since Tart still supports macOS 12.0 (Monterey) as a host.

Reply:
Oh, right, reverted that.